btconfig/utils/dataloader/ib.py

```diff
@@ -9,6 +9,8 @@
 from collections import defaultdict
 from datetime import datetime
 from time import sleep
+from time import time as timer
```

Comment:
why importing  time as timer? if time is going to be used, just import time from time